src/main/java/calculator/constants/Operator.java

```diff
@@ -15,13 +15,13 @@
     private String symbol;
     private DoubleBinaryOperator doubleBinaryOperator;
 
-    private static final Map<String, Operator> operatorCash;
+    private static final Map<String, Operator> OPERATOR_CASH;
```

Comment:
개인적인 생각입니다만..
OPERATOR_CASH가 캐시 역할을 하긴 하지만, 변수 자체가 담고 있는건 일종의 Operator의 집합이니까요.
```suggestion
    private static final Map<String, Operator> OPERATORS;
```
로 사용하는 건 어떨까요? 상수 및 인스턴스 변수의 이름을 적절히 관리해주는 것도 캡슐화의 일종이라고 생각이 드네요:)